4788/src/main/cpp/Robot.cpp

```diff
@@ -70,9 +70,8 @@ void Robot::TeleopInit() {
 }
 void Robot::TeleopPeriodic() {
 	shooter->teleopOnUpdate(dt);
```

Comment:
single tab slipped through the gaps. That's fine, we'll fix it in another pr

Reply:
FIxed